### PR TITLE
Fix the shield and the onBackPressed icons

### PR DESCRIPTION
**Pull request by makingthematrix**
#### APK
[Download build #10403](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10403/artifact/build/artifact/wire-dev-PR1441-10403.apk)
[Download build #10406](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10406/artifact/build/artifact/wire-dev-PR1441-10406.apk)
[Download build #10410](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10410/artifact/build/artifact/wire-dev-PR1441-10410.apk)
[Download build #10412](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10412/artifact/build/artifact/wire-dev-PR1441-10412.apk)
[Download build #10416](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10416/artifact/build/artifact/wire-dev-PR1441-10416.apk)
[Download build #10417](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10417/artifact/build/artifact/wire-dev-PR1441-10417.apk)
[Download build #10418](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10418/artifact/build/artifact/wire-dev-PR1441-10418.apk)
[Download build #10419](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10419/artifact/build/artifact/wire-dev-PR1441-10419.apk)
[Download build #10427](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10427/artifact/build/artifact/wire-dev-PR1441-10427.apk)

### CODE REVIEW

File: app/src/main/scala/com/waz/zclient/participants/ParticipantsController.scala [hunk 1]
```diff
@@ -94,6 +94,8 @@ class ParticipantsController(implicit injector: Injector, context: Context, ec:
 
   def getUser(userId: UserId): Future[Option[UserData]] = zms.head.flatMap(_.users.getUser(userId))
 
+  def userSignal(userId: UserId): Signal[UserData] = zms.flatMap(_.users.userSignal(userId))
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/participants/ParticipantsController.scala]: maybe unnecessary? I think unless the `ParticipantsController` is really doing anything to the user on it's way out of the SE, we should just go straight to the source

File: app/src/main/scala/com/waz/zclient/participants/fragments/GroupParticipantsFragment.scala [hunk 2]
```diff
@@ -108,6 +108,11 @@ class GroupParticipantsFragment extends FragmentHelper {
         .replace(R.id.fl__participant__container, new GuestOptionsFragment(), GuestOptionsFragment.Tag)
         .addToBackStack(GuestOptionsFragment.Tag)
         .commit
+
+      getParentFragment match {
+        case f: ParticipantFragment => f.setNavigationIconVisible(true)
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/participants/fragments/GroupParticipantsFragment.scala]: This should probably be done in the `GuestOptionsFragment`. It seems a bit unclean to be setting view state in the `onClick` handler that starts the fragment - what happens if we added another button to create the fragment? We'd have to remember to copy this code there as well.

**Comment by deanrobertcook** [comment]
build again

**Comment by deanrobertcook** [comment]
build again

**Comment by makingthematrix** [comment]
build again

**Pull request merged**